configs/j784s4_evm_hs_fs_all_defconfig

```diff
@@ -0,0 +1,13 @@
+# J784s4 EVM configuration for HS-FS devices
```

Comment:
Why are you deleting the configuration file and then introducing this file again. just introduce the AM69 SK file as a new file.

Reply:
Got it, will introduce a new file.. Thanks :slightly_smiling_face: